### PR TITLE
Extra return properties for LoadConfig.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Extra return properties for LoadConfig from K8s.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: https://github.com/dapr/dashboard/issues/103

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: pkg/runtime/cli.go [hunk 1]
```diff
@@ -155,7 +155,7 @@ func FromFlags() (*DaprRuntime, error) {
 			}
 			defer conn.Close()
 			namespace = os.Getenv("NAMESPACE")
-			globalConfig, configErr = global_config.LoadKubernetesConfiguration(*config, namespace, client)
+			globalConfig, _, configErr = global_config.LoadKubernetesConfiguration(*config, namespace, client)
```
Comment (youngbupark) [inline pkg/runtime/cli.go:158]: Why do we need to return original payload ?

Comment (artursouza) [reply pkg/runtime/cli.go:158]: Dashboard displays it. Otherwise, I cannot reuse this method.

Comment (artursouza) [reply pkg/runtime/cli.go:158]: Let me fix this, it needs to be in the standalone method, not this one.

Comment (artursouza) [reply pkg/runtime/cli.go]: See new version now, please.

**Review by yaron2 — APPROVED** [review]

**Pull request merged**